generator/nidaqmx_python_generator/metadata/attributes.py

```diff
@@ -0,0 +1,130 @@
+attributes = {
+    'Channel': {
+         6145: {
+            'access': 'read-write',
+            'name': 'AI_ATTEN',
+            'resettable': True,
+            'type': 'float64',
+            'ctypesDataType': 'ctypes.c_double',
+            'pythonDataType': 'float',
+            'description': 'Specifies the amount of attenuation to use.',
+            'hasExplicitReadBufferSize': False,
+            'bitFieldEnum':'N/A',
+            'isList':False,
+            'callingConvention':'StdCall',
+            'cFunctionName':'AIAtten',
+            'handleParameters':
+            {
+                'taskHandle':{
+                    "accessor": "self._handle", 
+                    "ctypesDataType": "lib_importer.task_handle", 
+                    "cviName": "taskHandle"
+                },
+                'ChannelHandle':{
```

Comment:
This is a name, not a handle.

Please decide whether these `handleParameters` keys are in camelCase, PascalCase, or snake_case.

Reply:
updated to use names